### PR TITLE
add-wait-for-clickable-function

### PR DESCRIPTION
**Pull request by FelixZilber**

### CODE REVIEW

**Comment by AlexeyCL** [comment]
@FelixZilber do you want to add tests?

File: src/commons/BrowserUtils.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ const CHILL_OUT_TIME: number = process.env.CHILL_OUT_TIME === undefined ? 3000 :
  * BrowserUtils wraps wdio browser functionality for cleaner test
  */
 export namespace BrowserUtils {
+  import WaitForOptions = WebdriverIO.WaitForOptions;
```
Comment (AlexeyCL) [inline src/commons/BrowserUtils.ts:21]: pls move to the beginning of the file

File: src/commons/BrowserUtils.ts [hunk 2]
```diff
@@ -991,6 +995,14 @@ export namespace BrowserUtils {
   export function rightClick(selector: string): void {
     Reporter.debug(`Right click mouse button on the element '${selector}'`);
     waitForEnabled(selector);
+
+    waitForClickable(selector);
+
     tryBlock(() => $(selector).click({ button: 'right' }), `Failed to preform right click on '${selector}'`);
   }
+
+  export function waitForClickable(selector: string, options?: WaitForOptions): void {
```
Comment (AlexeyCL) [inline src/commons/BrowserUtils.ts:1004]: pls add jdoc

File: src/commons/BrowserUtils.ts [hunk 3]
```diff
@@ -991,6 +995,14 @@ export namespace BrowserUtils {
...
+    Reporter.debug('Wait for element to be clickable');
```
Comment (AlexeyCL) [inline src/commons/BrowserUtils.ts:1005]:
```suggestion
    Reporter.debug(`Wait for the element ${selector} to be clickable`);
```

**Review by AlexeyCL — APPROVED** [review]

File: src/commons/BrowserUtils.ts (hunk 1 above)
Comment (FelixZilber) [reply src/commons/BrowserUtils.ts:21]: good catch!

File: src/commons/BrowserUtils.ts (hunk 2 above)
Comment (FelixZilber) [reply src/commons/BrowserUtils.ts:1004]: Done

File: src/commons/BrowserUtils.ts (hunk 3 above)
Comment (FelixZilber) [reply src/commons/BrowserUtils.ts:1005]: Good idea. Thanks

**Pull request merged**